packages/block-editor/src/style.scss

```diff
@@ -37,3 +37,11 @@
 @import "./components/warning/style.scss";
 @import "./components/writing-flow/style.scss";
 @import "./hooks/anchor.scss";
+
+body {
+	@include text-body-small;
```

Comment:
@davewhitley Halloo!!! This PR looks great :).

The thing that stuck out to me was applying styles to the overall `body`. For a plugin-like thing like Gutenberg, it feels strange that it would target something so broad outside of itself.

For example, when this is loaded in WP-Admin, I'm not sure what `body p` may affect.

If we wanted to adjust "global" (in the context of Gutenberg) font styles, would we be able to target just Gutenberg?

Hope that makes sense!

Reply:
If we could specify Gutenberg only instead of `body` that'd be great (there's probably a good way to do that). I imagine it would take a few months to change those global styles in WP Admin.

Reply:
Just checked locally. It looks like the styles are being overridden anyway by the existing `common.css` 😅 

<img width="870" alt="Screen Shot 2019-11-14 at 08 37 19" src="https://user-images.githubusercontent.com/2322354/68836193-138e5480-06ba-11ea-8d88-9c33609831d3.png">

Reply:
To apply things **just** to the editor, it looks like we have to update this file:
https://github.com/WordPress/gutenberg/blob/master/packages/edit-post/src/style.scss#L65

Adding the `@include text-body-small` mixin there appeared to work 👍